Frontend/ui-driver/src/Services/APITypes.purs

```diff
@@ -1202,3 +1203,218 @@ instance showGetGetPerformanceRes :: Show GetPerformanceRes where show = generic
 instance standardEncodeGetPerformanceRes :: StandardEncode GetPerformanceRes where standardEncode (GetPerformanceRes req) = standardEncode req
 instance decodeGetPerformanceRes :: Decode GetPerformanceRes where decode = defaultDecode
 instance encodeGetPerformanceRes :: Encode GetPerformanceRes where encode = defaultEncode
+
+--------------------------------------------------- getCategories ----------------------------------------------------
+data GetCategoriesReq = GetCategoriesReq String
+
+newtype GetCategoriesRes = GetCategoriesRes
+  { categories :: Array
```

Comment:
@ThisIsMani @mohdshoaib18169 Can you create another newtype and then use instead of adding object directly